CONTRIBUTORS.txt

```diff
@@ -0,0 +1,161 @@
+# This file is autogenerated by 'contributors-txt',
+# please do not modify manually
+
+Contributors
+------------
+- Claudiu Popa <pcmanticore@gmail.com>
+- LOGILAB S.A. (Paris, FRANCE) <contact@logilab.fr>
+- Sylvain Thénault <thenault@gmail.com>
+- Pierre Sassoulas <pierre.sassoulas@gmail.com>
+- hippo91 <guillaume.peillex@gmail.com>
+- Marc Mueller <30130371+cdce8p@users.noreply.github.com>
+- Daniël van Noord <13665637+DanielNoord@users.noreply.github.com>
+- Google, Inc. <None>
```

Comment:
Should we keep `<None>` or remove it in this case?

Reply:
Nice catch !